Libplanet.Tests/Tx/TxIdTest.cs

```diff
@@ -231,6 +193,28 @@ public void Compare()
             Assert.Throws<ArgumentException>(() => txIds[0].CompareTo("invalid"));
         }
 
+        [Fact]
+        public void Bencoded()
+        {
+            var expected = new TxId(TestUtils.GetRandomBytes(TxId.Size));
+            var deserialized = new TxId(expected.Bencoded);
+            Assert.Equal(expected, deserialized);
+            expected = default(TxId);
+            deserialized = new TxId(expected.Bencoded);
+            Assert.Equal(expected, deserialized);
```

Comment:
```suggestion
            var expected2 = default(TxId);
            var deserialized2 = new TxId(expected.Bencoded);
            Assert.Equal(expected2, deserialized2);
            Assert.NotEqual(expected, deserialized2);
```

This might help if Bencoded implementation is unintentionally changed to return something constant.

Reply:
I understand the precaution, but what does this achieve exactly?
If `Bencoded` implementation gets broken to return something constant, the test for encoding and decoding the first random `TxId` would fail anyways. 😶

Reply:
You're right, in the hypothetical situation it should fail on the first `Assert.Equal()`. I thought the precautionary test might flag on something, but I can't really come up with something that makes sense. 😞